### PR TITLE
Added Nmapper Payload

### PR DESCRIPTION
**Pull request by thehappydinoa**
Added Nmaper Payload similar to @hak5darren 's for the Bash Bunny

### CODE REVIEW

**Comment by sebkinne** [comment]
Did you have a chance to test this payload? I'm not sure if this will work in TRANSPARENT mode, because the PS doesn't receive an IP, but I haven't had a chance to test it.

**Comment by thehappydinoa** [comment]
I haven't been able to test it, I would love if someone could though.

**Comment by sundhaug92** [comment]
If GATEWAY_* isn't set in TRANSPARENT-mode, you could change it to a BRIDGE-mode payload. Alternatively, a scapy-script should be able to infer the gateway by capturing traffic (preferably DHCP).

**Comment by thehappydinoa** [comment]
@sundhaug92 Thats a good idea.

File: payloads/library/recon/nmaper/payload.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Title:         Nmaper
+# Description:   Nmap's the entire network
+# Author:        thehappydinoa
+# Version:       1.0
+# Category:      Recon
+# Target:        Any
+# Net Mode:      TRANSPARENTs
```
Comment (sebkinne) [inline payloads/library/recon/nmaper/payload.sh]: This should be TRANSPARENT

**Comment by thehappydinoa** [comment]
@sebkinne I fixed the typo in payload.sh

**Comment by thehappydinoa** [comment]
@sebkinne What does it mean that its approved but not merged?

**Comment by sebkinne** [comment]
@thehappydinoa I just approved the changes you made after I requested them. I haven't merged the payload yet because I haven't had time to test it, and haven't seen reports of it being tested on the forums either.

**Comment by thehappydinoa** [comment]
@sebkinne Ah

**Comment by sundhaug92** [comment]
Re 9a1b5d0, typo persists in dir-name

**Pull request closed without merging**